internal/venice-common/src/test/java/com/linkedin/venice/controllerapi/RequestTopicForPushRequestTest.java

```diff
@@ -0,0 +1,109 @@
+package com.linkedin.venice.controllerapi;
+
+import static com.linkedin.venice.meta.Version.PushType.BATCH;
+import static org.mockito.Mockito.*;
```

Comment:
nit: * import

Reply:
Thanks for catching it. Fixed this and several other places.